app/serializers/supplejack_api/mlt_serializer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module SupplejackApi
+  class MltSerializer < ActiveModel::Serializer
+    attribute :id
+
+    RecordSchema.fields.each do |name, definition|
+      if definition.search_value.present? && definition.store == false
+        attribute name do
+          definition.search_value.call(object)
+        end
+      else
+        attribute name do
+          if object.public_send(name).nil?
+            definition.default_value
+          elsif definition.date_format.present?
+            format_date(object.public_send(name), definition.date_format)
+          else
+            object.public_send(name)
+          end
+        end
+      end
+    end
+
+    private
+
+    def format_date(date, format)
+      date.strftime(format)
+    end
```

Comment:
is that a very useful method?

Reply:
Maybe not at this point i can just remove it with simple formatting for now